demos/classification_benchmark_demo/cpp/main.cpp

```diff
@@ -26,90 +26,93 @@
+
+constexpr char auto_resize_msg[] = "enables resizable input";
+DEFINE_bool(auto_resize, false, auto_resize_msg);
+
+constexpr char d_msg[] = "specify the target device to infer on. "
+    "The list of available devices is shown below. The demo will look for a suitable plugin for device specified. "
+    "Default value is CPU";
+DEFINE_string(d, "CPU", d_msg);
+
+constexpr char gt_msg[] = "path to ground truth .txt file";
+DEFINE_string(gt, "", gt_msg);
+
+constexpr char layout_msg[] = "specify inputs layouts."
+    "Ex. \"[NCHW]\" or \"input1[NCHW],input2[NC]\" in case of more than one input";
+DEFINE_string(layout, "", layout_msg);
+
+constexpr char nireq_msg[] = "number of infer requests";
+DEFINE_uint32(nireq, 0, nireq_msg);
+
+constexpr char nstreams_msg[] = "specify count of streams";
+DEFINE_string(nstreams, "", nstreams_msg);
+
+constexpr char nt_msg[] = "number of top results. Must be >= 1. Default value is 5";
+DEFINE_uint32(nt, 5, nt_msg);
+
+constexpr char nthreads_msg[] = "specify count of threads";
+DEFINE_uint32(nthreads, 0, nthreads_msg);
+
+constexpr char res_msg[] = "set image grid resolution in format WxH. Default value is 1280x720";
+DEFINE_string(res, "1280x720", res_msg);
+
+constexpr char show_msg[] = "disable showing of processed images";
+DEFINE_bool(show, true, show_msg);
+
+constexpr char time_msg[] = "time in seconds to execute program. Default is -1 (infinite time)";
+DEFINE_uint32(time, std::numeric_limits<gflags::uint32>::max(), time_msg);
+
+constexpr char u_msg[] = "list of monitors to show initially";
+DEFINE_string(u, "", u_msg);
+
+void parse(int argc, char *argv[]) {
+    gflags::ParseCommandLineFlags(&argc, &argv, false);
+    if (FLAGS_h || 1 == argc) {
+        std::cout <<   "\t[ -h]                 " << h_msg
+                << "\n\t  -i <INPUT>            " << i_msg
+                << "\n\t --labels <LABELS>     " << labels_msg
+                << "\n\t  -m <MODEL FILE>       " << m_msg
+                << "\n\t[--auto_resize]         " << res_msg
+                << "\n\t[ -d <DEVICE>]          " << d_msg
+                << "\n\t[--gt <STRING>]       " << gt_msg
+                << "\n\t[--layout <STRING>] " << layout_msg
+                << "\n\t[--nireq <NUMBER>]      " << nireq_msg
+                << "\n\t[--nstreams <NUMBER>]   " << nstreams_msg
+                << "\n\t[--nt <NUMBER>]         " << nt_msg
+                << "\n\t[--nthreads <NUMBER>]   " << nthreads_msg
+                << "\n\t[--res <STRING>]       " << res_msg
+                << "\n\t[--show] ([--noshow])   " << show_msg
+                << "\n\t[--time <NUMBER>]       " << time_msg
+                << "\n\t[ -u]                   " << u_msg
+                << "\n\tKey bindings:"
+                    "\n\t\tQ, q, Esc - Quit"
+                    "\n\t\tR, r, SpaceBar - Restart testing"
+                    "\n\t\tC - average CPU load, D - load distrobution over cores, M - memory usage, H - hide\n";
```

Comment:
You should revise code here according to PR with interactive face detection demo, since there might be more copy-pasted mistakes like this one.
```suggestion
                    "\n\t\tC - average CPU load, D - load distribution over cores, M - memory usage, H - hide\n";
```